packages/frontend-api/src/Resources/config/graphql-types/ModelType/Watchdog/Input/CreateWatchdogInputDecorator.types.yaml

```diff
@@ -0,0 +1,21 @@
+CreateWatchdogInputDecorator:
+    type: input-object
+    decorator: true
+    config:
+        validation:
+            - Shopsys\FrontendApiBundle\Component\Constraints\Watchdog: ~
```

Comment:
this may be simpler field validation as it only validate the data from one field (productUuid)